syntax = "proto3";

package example;

option cc_generic_services = true;

message EchoRequest {
    bytes message = 1;
}

message EchoResponse {
    bytes message = 1;
}

service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}